assets/scss/_base.scss

```diff
@@ -832,3 +832,16 @@ section#cncf {
     font-size: 1rem;
   }
 }
+
+/* DOCUMENTATION */
```

Comment:
@sftim What do you think about adding a comment at the top of the file that points to this section? 

I'm concerned that in a file with hundreds of lines and without otherwise prevalent comments, this setting may be lost/opaque to our 90+ days-in-the-future selves. 😅 

Same for `_custom.scss` below.

Reply:
I'm game, but as the author the way this works is obvious _to me_.
What should the comment say?

Reply:
> What should the comment say?

Eh, that’s fair.

~“This cascading style sheet does, in fact, cascade.”~

I withdraw the suggestion. 😂